js/bitmap-map.js

```diff
@@ -110,7 +108,7 @@ function setStyle(style)
 	// helper variable for saving current map style
 	railmap.selectedStyle = style;
 	// change tileserver url to load different style
-	railmap._url = tiledir+style+'/{z}/{x}/{y}.png';
+	railmap._url = window.openrailwaymap.tiledir+style+'/{z}/{x}/{y}.png';
```

Comment:
when you touch the line anyway, how about some spaces around the operators?

Reply:
Instead of fixing coding style in this pull request, I would propose to do this later by integrating a tool like ESLint in out workflow.

Reply:
Possible. I tend to fix such things in lines I touch anyway. YMMV.